src/test/java/com/klarna/hiverunner/sql/StatementsSplitterTest.java

```diff
@@ -160,4 +161,16 @@ public void testReadUntilEndOfLine() {
         Assert.assertEquals("", StatementsSplitter.readUntilEndOfLine(tokenizer));
     }
 
+    @Test
+    public void beelineSqlLineCommandsAreSupported() {
+        String statementA = "!run script.hql";
+        String statementB = "select * from table where foo != bar";
+        String statementC = "!run another_script.hql";
+
+        List<String> expected = Arrays.asList(statementA, statementB, statementC);
+        String expression = statementA + '\n' + statementB + ";   " + statementC;
+
+        Assert.assertEquals(expected, StatementsSplitter.splitStatements(expression));
+    }
+
 }
```

Comment:
I noticed that `expression` already adds whitespace before `statementC` to verify trimming. Do you think it's OK?